src/pages/addCard/components/addCard/AddCardForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { useAddCardForm } from '@pages/addCard/hooks';
+import {
+	CardNumberInput,
+	CardExpiredDateInput,
+	CardHolderNameInput,
+	CardSecurityCodeInput,
+	CardPasswordInput,
+} from '@pages/addCard/components';
+import { Button } from '@components/ui-kit';
+import { PaymentCard } from '@components/domain';
+
+export default function AddCardForm() {
+	const {
+		displayedCardNumber,
+		displayedExpiredDate,
+		cardHolderName,
+		cardSecurityCode,
+		firstCardPassword,
+		secondCardPassword,
+		handleCardNumberChange,
+		handleCardExpiredDateChange,
+		handleCardHolderNameChange,
+		handleCardSecurityCodeChange,
+		handleFirstCardPasswordChange,
+		handleSecondCardPasswordChange,
+		handleSubmit,
+	} = useAddCardForm();
+
+	return (
+		<form onSubmit={handleSubmit}>
+			<PaymentCard
+				cardNumber={displayedCardNumber}
+				cardExpiredDate={displayedExpiredDate}
+				cardHolderName={cardHolderName}
+			/>
+			<CardNumberInput
+				value={displayedCardNumber}
+				onChange={handleCardNumberChange}
+			/>
+			<CardExpiredDateInput
+				value={displayedExpiredDate}
+				onChange={handleCardExpiredDateChange}
+			/>
+			<CardHolderNameInput
+				value={cardHolderName}
+				onChange={handleCardHolderNameChange}
+			/>
+			<CardSecurityCodeInput
+				value={cardSecurityCode}
+				onChange={handleCardSecurityCodeChange}
+			/>
+			<CardPasswordInput
+				firstValue={firstCardPassword}
+				secondValue={secondCardPassword}
+				onFirstValueChange={handleFirstCardPasswordChange}
+				onSecondValueChange={handleSecondCardPasswordChange}
+			/>
```

Comment:
```suggestion
	const {
		cardProps, 
		numberProps, 
		expiredDateProps, 
		holderNameProps, 
		securityCodeProps, 
		passwordProps
	} = useAddCardForm();

	return (
		<form onSubmit={handleSubmit}>
			<PaymentCard {...cardProps} />
			<CardNumberInput {...numberProps} />
			<CardExpiredDateInput {...expiredDateProps} />
			<CardHolderNameInput {...holderNameProps} />
			<CardSecurityCodeInput {...securityCodeProps} />
			<CardPasswordInput {...passwordProps} />
```

이런식으로 표현하면 더 깔끔하지 않을까요!?